server/src/main/java/org/apache/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -488,6 +488,116 @@ private boolean isRetryTimeout(long now, long timeout, long beginTime) {
         return timeout >= ALWAYS_RETRY_BOUNDARY && now - beginTime > timeout;
     }
 
+    /**
+     * Handle rollbacking by scheduled.
+     */
+    protected void handleRollbackingByScheduled() {
+        long delay = ROLLBACKING_RETRY_PERIOD;
+        SessionCondition sessionCondition = new SessionCondition(rollbackingStatuses);
+        sessionCondition.setLazyLoadBranch(true);
+        List<GlobalSession> rollbackingSessions =
+            SessionHolder.getRootSessionManager().findGlobalSessions(sessionCondition);
+        if (CollectionUtils.isEmpty(rollbackingSessions)) {
+            rollbackingSchedule(delay);
+            return;
+        }
+        rollbackingSessions.sort(Comparator.comparingLong(GlobalSession::getBeginTime));
+        List<GlobalSession> needDoRollbackingSessions = new ArrayList<>();
+        for (GlobalSession rollbackingSession : rollbackingSessions) {
+            long time = rollbackingSession.timeToDeadSession();
+            if (time <= 0) {
+                needDoRollbackingSessions.add(rollbackingSession);
+            } else {
+                delay = Math.max(time, ROLLBACKING_RETRY_PERIOD);
```

Comment:
这里应该记录delay，做break即可，然后foreach后的时间直接用这个时间，否则可能会出现多个定时任务在同时运行
The delay should be recorded here, just do break, and then use this time directly after foreach, otherwise multiple scheduled tasks may run at the same time